SEED-platform/seed
# Feat/column settings

#### Any background context you want to provide?
Continuation of analysis stats for summary tab

#### What's this PR do?
Included a table with columns through http API (not showing up in the summary tab on the Inventory page yet).  Need to add in the counts that exist for each column still.  This is for the properties only (not tax lots)

#### How should this be manually tested?
Through API http (see example below).  Fill in org id and cycle id to run
http://0.0.0.0:8000/api/v3/analyses/stats/?organization_id=1&cycle_id=1

#### What are the relevant tickets?
 Create plots for column-based statistics #2741 

#### Screenshots (if appropriate)
example of how first few columns in table look today
<img width="504" alt="Screen Shot 2021-08-20 at 9 21 52 AM" src="https://user-images.githubusercontent.com/48136858/130256014-c5aa2c82-3b7a-40d4-a577-6f0368afa8ea.png">
